src/Altinn.App.PlatformServices/Implementation/ApplicationLanguage.cs

```diff
@@ -36,52 +36,28 @@ public class ApplicationLanguage : IApplicationLanguage
         /// <inheritdoc />
         public async Task<List<Platform.Storage.Interface.Models.ApplicationLanguage>> GetApplicationLanguages()
         {
-            string pathTextsResourceFolder = _settings.AppBasePath + _settings.ConfigurationFolder + _settings.TextFolder;
-
-            DirectoryInfo directoryInfo = new DirectoryInfo(pathTextsResourceFolder);
+            var pathTextsResourceFolder = Path.Join(_settings.AppBasePath, _settings.ConfigurationFolder, _settings.TextFolder);
+            var directoryInfo = new DirectoryInfo(pathTextsResourceFolder);
 
             if (!directoryInfo.Exists)
```

Comment:
You don't really need this check either since you now catch DirectoryNotFoundException in the controller.

Reply:
✔️